### DISCUSSION TITLE
Use a seperate repo for docs' translations

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [vuejs/pinia#2097] [Ideas]
I notice that there are many volunteers want to add translations for docs. It will be hard to maintain.

**Comment by posva** [comment]
So far it seems fine. The main issue is the API docs are generated from the source code. Not sure if that will work when using a different repository

**Comment by awxiaoxian2020** [comment]
@posva The en docs are generated from the source code indeed. But the zh docs seem only a copy of en docs even now. It seems not to relate to source code. We can re-build using vitepress with locale doc contents.
